python/cxx/eclipse_io.cpp

```diff
@@ -33,6 +37,105 @@ py::array get_vector(Opm::EclIO::EclFile * file_ptr, std::size_t array_index) {
     throw std::logic_error("Data type not supported");
 }
 
+bool erst_contains(Opm::EclIO::ERst * file_ptr, std::tuple<std::string, int> keyword) {
+
+    bool hasKeyAtReport = file_ptr->count(std::get<0>(keyword), std::get<1>(keyword)) > 0 ? true : false;
+    
```

Comment:
The use of blank lines should help the reader understand the structure in the code - your blank seems to be spread out a bit randomly?